lib/auth/auth_with_roles_test.go

```diff
@@ -3771,3 +3771,113 @@ func TestGetActiveSessionTrackers(t *testing.T) {
 		})
 	}
 }
+
+func TestListReleasesPermissions(t *testing.T) {
+	ctx := context.Background()
+	srv := newTestTLSServer(t)
+
+	tt := []struct {
+		Name         string
+		Role         types.RoleSpecV5
+		ErrAssertion func(*testing.T, error)
```

Comment:
Nit, refactor

since each ErrAssertion value is:
`ErrAssertion: func(t *testing.T, err error) { require.<BOOL>(t, trace.IsAccessDenied(err)) },`

You can set this as:
`ErrAssertion require.BoolAssertionFunc`
Pass it as:
`ErrAssertion: require.False,` - or .True
and assert as:
`tc.ErrAssertion(t, trace.IsAccessDenied(err))`